src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    Red,

```

Comment:
Enum values should be in uppercase, separated by underscores if needed. For example, RED instead of Red. Please update the enum values accordingly. 📝